### PR TITLE
add blog entry for Apache NetBeans 23

### PR DESCRIPTION
**Pull request by ebarboni**

### CODE REVIEW

File: modules/ROOT/pages/blogs/entry/announce-apache-netbeans-23-released.adoc [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+:page-syntax: true
+
+
+The Apache NetBeans team is pleased to announce that Apache NetBeans 23 was released on September 19, 2024.
+
+What's in the Apache NetBeans 23 release:
+
+https://github.com/apache/netbeans/releases/tag/23
+
+With thanks to 22 contributors, including 3 who have contributed for the first time. Thank you!
```
Comment (mbien) [inline modules/ROOT/pages/blogs/entry/announce-apache-netbeans-23-released.adoc]: its 24 contributors according to the release notes page

File: modules/ROOT/pages/blogs/entry/announce-apache-netbeans-23-released.adoc [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+//     Unless required by applicable law or agreed to in writing,
+//     software distributed under the License is distributed on an
+//     "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+//     KIND, either express or implied.  See the License for the
+//     specific language governing permissions and limitations
+//     under the License.
+//
+
+= [ANNOUNCE] Apache NetBeans 23 Released
+:author: Geertjan Wielenga
```
Comment (mbien) [inline modules/ROOT/pages/blogs/entry/announce-apache-netbeans-23-released.adoc]: change to your name?

Comment (ebarboni) [reply modules/ROOT/pages/blogs/entry/announce-apache-netbeans-23-released.adoc]: ok

File: modules/ROOT/pages/blogs/entry/announce-apache-netbeans-23-released.adoc (hunk 1 above)
Comment (ebarboni) [reply modules/ROOT/pages/blogs/entry/announce-apache-netbeans-23-released.adoc]: done

**Pull request merged**